### ISSUE TITLE
evm utils cannt find keypair at custom path

### ISSUE THREAD

**Issue by vldm** [velas/velas-chain#164]
#### Problem

```
root@velas-solana-bridge-68696ffcfb-z6s8v:/# evm-utils -r localhost:8899 transfer-to-evm 1000 0xacc02d07a5f51070c705cde1ebb35142b3f059c8
[2021-07-08T12:33:15Z INFO  evm_utils] Loading keypair from: /root/.config/velas/id.json
thread 'main' panicked at 'called Result::unwrap() on an Err value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', evm-utils/src/main.rs:92:55
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
```

**Comment by roman-tik** [comment]
+ make other parameters (e.g. --url/-u instead of -r) inline with other cli utils

**Comment by vldm** [comment]
Closed in #183
